nservicebus/upgrades/5to6.md

```diff
@@ -654,13 +654,22 @@ The `NServiceBus.Callbacks` package has to be referenced only by the requesting
 
 snippet: 5to6-Callbacks
 
-In Version 6 the callback handling on [broker transports](/nservicebus/transports/#types-of-transports-broker-transports) is no longer based on the machine host name. In order to continue using callbacks when using a broker transport, a unique endpoint instance ID has to be provided:
+In Version 6 the callback handling on [broker transports](/nservicebus/transports/#types-of-transports-broker-transports) is no longer based on the machine host name. In order to continue using callbacks when using a broker transport, the endpoint needs to be uniquely addressable:
 
 snippet: 5to6-Callbacks-InstanceId
 
 NOTE: This ID should never be hardcoded, e.g. it can be read from the configuration file or from the environment (e.g. role ID in Azure), so that it can be changed without code changes and redeployment.
 
 
+## Uniquely addressable endpoint instances
```

Comment:
This is already covered in the callbacks doco and in the Azure data distribution doco. I would remove this section because it is not directly related to he upgrade process.